.github/CODEOWNERS

```diff
@@ -1,14 +1,7 @@
 #
 # docs, prose, legal
 #
-licenses           @senior7515 @dswang
-/CONTRIBUTING.md   @dotnwat
-
-#
-# editors
-#
-/.vim             @dotnwat
-/.dir-locals.el   @graphcareful
+licenses           @emaxerrno @dswang
```

Comment:
Changes:
- Alex changed his username (https://redpandadata.slack.com/archives/CPKP64GDN/p1656373689863429)
- removed @dotnwat and @graphcareful as code owners since I believe everyone changing these can find a reviewer (at least by finding who was the person with most commits).

If we want to keep some of these, here is a proxy metric to who should be owning these (number of commits):
```
$ git shortlog -s -n --all CONTRIBUTING.md 
     3  Alexander Gallego
     2  David Castillo
     2  Rob Blafford
```
```
$ git shortlog -s -n --all .vim/
     2  Noah Watkins
```
```
$ git shortlog -s -n --all .dir-locals.el
     7  Alexander Gallego
     2  Rob Blafford
     1  Andrew Hsu
     1  Kostas Kyrimis
```